trishalanglois/rancid-tomatillos
# Component setup gi

### What does this PR do?
- Sets up components for: `MovieContainer`, `Nav`,` Movie`
- Create Redux cycle for `addMovie`
- Fetch the data 
- Install `react-router`
- Display the movies in the `moviesContainer`

### How can you test?
- Pull down these changes and take a look at App.js, you can see that there is now a Nav/MoiveContainer component being rendered. Within the MovieContainer we are rendering our movie cards. You can see everything is hooked up correct when the movies are displaying on the page. Another thing to check will be the movies inside our Redux store, open the Redux dev tools and see them inside...

### Issues:
- This PR will close all issues pertaining from `Iteration 1`

### Notes 
- There might be a chance to refactor the below render of the loading message into a component..
<img width="491" alt="Screen Shot 2019-12-18 at 1 55 13 PM" src="https://user-images.githubusercontent.com/48968224/71122522-07585400-219e-11ea-94e4-cc86d63e8283.png">

Thanks!

@trishalanglois  @sertmer
